draft-ietf-rats-corim.md

```diff
@@ -91,17 +91,18 @@ appraisal of Evidence. Therefore that burden is typically offloaded to a
 Verifier.  In order to conduct Evidence appraisal, a Verifier requires not only
 fresh Evidence from an Attester, but also trusted Endorsements and Reference
 Values from Endorsers and Reference Value Providers, such as manufacturers,
-distributors, or device owners.  This document specifies Concise Reference
-Integrity Manifests (CoRIM) that represent Endorsements and Reference Values in
-CBOR format.  Composite devices or systems are represented by a collection of
-Concise Module Identifiers (CoMID) and Concise Software Identifiers (CoSWID)
-bundled in a CoRIM document.
+distributors, or device owners. This document specifies the information elements  for conveying Endorsed Values and Reference Values as CBOR data that shall be used by Supply Chain Entities namely Endorsed Value & Reference Value providers.
 
 --- middle
 
 # Introduction
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/4
+Supply Chain entities provide the necessary Endorsed and Reference Values to a Verifier. Some examples of supply chain entites are original equipement manufacturers (OEMs), original device manufactueres (ODMs), firmware vendors, software vendors etc. Generally, these Values are information provided by the manufacturer and suppliers about the Attester.  In a complex supply chain, it is likely multiple entities will produce these values at different point in time.
+
+This document defines the schema for conveying the Endorsed Values and Reference Values to a Verifier. The schema defined in this document facilitate interoperability between different supply chain entities and also between multiple Verifiers receiving these Endorsed and Reference Values.
+
+This document specifies Concise Reference Integrity Manifests (CoRIM) that represent top level schema for expressing Endorsements and Reference Values in
+CBOR format. Inside CoRIM, Concise Module ID (CoMID) tags are used to express metadata and claims about the hardware and firmware modules. CoMID can be used to describe platforms, devices and components. A SWID Tag models software asset management. A CoSWID Tag represents a Concise representation of a SWID tag and can be implemented in a memory efficient manner. A CoRIM consists of one or more CoMID and/or CoSWID tags.
```

Comment:
'CBOR format.' should be moved to line 104.
'tags are used to express' seems verbose : suggest "tags contain metadata and Claims pertaining to hardware and firmware.'
Suggest: "Comid tags can describe..."
Suggest: "SWID tags can describe software..."
'Concise' to 'concise'
'SWID' should include a reference to [ISO_SWID]
Suggest: "...SWID tag that can be..."
The last sentence should be revised to say: "A CoRIM may contain multiple tags and can be extended to include different tag types. The base CoRIM schema defines three tag types: CoMID, CoSWID, and CoBOM."